src/ring/middleware/oauth2.clj

```diff
@@ -41,16 +41,29 @@
   (-> {:token access_token}
       (cond-> expires_in (assoc :expires (-> expires_in time/seconds time/from-now)))))
 
-(defn- get-access-token [{:keys [access-token-uri client-id client-secret]} request]
+(defn- request-params
+  [profile request]
```

Comment:
Can you ensure the arguments to the function are on the same line as the `defn` if there's space?